README.md

```diff
@@ -10,15 +10,15 @@ It made the children laugh and play,
 And so the Teacher turned him out,  
    But still he slithered near,  
 And waited patiently about,  
-   Till Mary did appear ;  
+   Till Walt did appear ;  
```

Comment:
These weird spaces before punctuation are bothering me. Let's fix these.

Reply:
I agree, I'll update the PR